tern/formats/json/html.py

```diff
@@ -0,0 +1,175 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2020 VMware, Inc. All Rights Reserved.
+# SPDX-License-Identifier: BSD-2-Clause
+
+from tern.utils import general
+
+def get_version():
+    '''Return the version string for the --version command line option'''
+    ver_type, commit_or_ver = general.get_git_rev_or_version()
+    message = ''
+    if ver_type == "package":
+        message = "Tern version {}".format(commit_or_ver)
+    else:
+        message = "Tern at commit {}".format(commit_or_ver)
+    return message
```

Comment:
This function is unnecessary --`tern/report/content.py` has a function called `get_tool_version()` that will already do this work for you:

```
 15 def get_tool_version():                                                         
 16     '''Return a string describing the version and where it came from'''         
 17     ver_type, ver = get_git_rev_or_version()                                    
 18     if ver_type == 'commit':                                                    
 19         return formats.commit_version.format(commit_sha=ver)                    
 20     return formats.packaged_version.format(version=ver)    
```

So you can